example_notebooks/6_mls_wheat_demo.ipynb

```diff
@@ -0,0 +1,2666 @@
+{
```

Comment:
<p>Here, we should add that the same object (the wheat plant) is placed in the scene 15 times. Using the translate filter, it is positioned in three columns and five rows to form a wheat field.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>